python/ray/data/dataset.py

```diff
@@ -1829,7 +1829,7 @@ def iter_rows(self, *, prefetch_blocks: int = 0) -> Iterator[T]:
             A local iterator over the entire dataset.
         """
         for batch in self.iter_batches(
-                prefetch_blocks=prefetch_blocks, batch_format="native"):
```

Comment:
Hmm if a user was trying to keep everything in Arrow, this hard-coded Pandas batch would break that. Could we expose this in the `.iter_rows()` API as `row_format`, with a default to `"pandas"`?

Reply:
Good point!